plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/AbstractKuduConnectorTest.java

```diff
@@ -276,6 +276,23 @@ public void testCreateTable()
         throw new SkipException("TODO");
     }
 
+    @Override
+    public void testCreateTableWithTableComment()
+    {
+        String tableName = "test_create_" + randomTableSuffix();
+        String catalogName = getSession().getCatalog().orElseThrow();
+        String schemaName = getSession().getSchema().orElseThrow();
+
+        assertUpdate("CREATE TABLE " + tableName + "(" +
+                "id INT WITH (primary_key=true)," +
+                "a VARCHAR" +
+                ")" +
+                "COMMENT 'test comment'" +
+                " WITH (partition_by_hash_columns = ARRAY['id'], partition_by_hash_buckets = 2)");
+        assertEquals(getTableComment(catalogName, schemaName, tableName), "test comment");
+        assertUpdate("DROP TABLE IF EXISTS " + tableName);
+    }
```

Comment:
Please fix `testCreateTableAsSelectWithTableComment()` as well.

Reply:
@ebyhr Is there a specific reason to override that test? I have found that a simple fix has helped many of the tests pass including that test.

Reply:
@ebyhr How should I go about overriding this test? 
The 3 tests that failed for `testCreateTableAsSelectWithTableComment()` was due to the earliest tag version (`1.13.0`) of Kudu not supporting create table with table comments. 
We can only support create table with table comments for Kudu tag versions `1.15.0` and above. 

The same issue for the 3 tests that failed for `testCreateTableWithComment()` that used that earlier tag version as well.

Reply:
You can use different assertions per `getKuduServerVersion()`.
```java
if (getKuduServerVersion is EARLIEST_TAG) {
 ...
}
else {
 ...
}
```

By the way, the tag is server version, not client version.